### ISSUE TITLE
How to ignore routes that do not start with certain specific prefixes

### ISSUE THREAD

**Issue by Red-Asuka** [harlan-zw/nuxt-simple-sitemap#91]
### Details

My site has multiple languages, and the i18n strategy is set to prefix. However, there is a bug with this i18n setting, where routes without a prefix are still added to the route table. Therefore, I want to exclude all routes that do not start with a language prefix. In the documentation, there is an example of how to use "exclude" (https://nuxtseo.com/sitemap/api/config#exclude). I tried using the following settings to block routes that do not start with a language prefix, but it didn't work. Could you please tell me the correct settings?

```ts
  exclude: [
    /^\/(?!en|zh|ja).*/,
  ],
```

**Comment by harlan-zw** [comment]
Hey @Red-Asuka, thanks for the issue.

I've noticed a bug with using the `prefix` strategy and have pushed up a fix in v3.0.11.

If you still have issues, I'm happy to try and solve the underlying issues. I just need some reproduction details.

To answer your question though, you need to use the same patterns as route rules. You would need to combine this with the `include` rule as there's no way to do the negative look ahead.

```ts
  include: [
    '/en/**',
    '/zh'**',
    // etc
  ],
```

I'll need to update the docs to explain that better.

**Comment by Red-Asuka** [comment]
@harlan-zw Hello, I have upgraded the version to v3.0.12. All pages still have routes without language prefixes, and all prefix routes except for the default language (en) are gone. This is the minimal replication: https://stackblitz.com/edit/github-fig8fh?file=nuxt.config.ts

<img width="519" alt="image" src="https://github.com/harlan-zw/nuxt-simple-sitemap/assets/38158783/ec7022d6-dada-4f73-8cc3-c47e8c61f99c">

**Comment by harlan-zw** [comment]
Thanks for the reproduction!

This was actually an interesting bug. The problem was that I hadn't coded the integration to work when you use inline options for i18n within the nuxt.config.

I sent two Nuxt PRs to address this:
- https://github.com/nuxt/nuxt/pull/22316
- https://github.com/nuxt/nuxt/pull/22317

Anyway fix is available in 3.0.13
